### DISCUSSION TITLE
How to write attractive/effective readmes for projects

### DISCUSSION THREAD

**Discussion by ml-runner** [community/community#20616] [Other Feature Feedback, Questions, & Ideas] [answered]
How can one make their project likeable and attractive , how to write a cool looking and organised readme?

**Comment by prathikshetty2002** [comment, marked as answer]
To create an effective and good looking Readme markdown you can read this blog by freecodecamp: [How to writre a good readme](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/ ) 

Also if you need to generate a readme file directly and not type the markdown use [readme.so](https://readme.so/)

Hope this helps, mark this as an answer :)
